todo.go

```diff
@@ -4,13 +4,54 @@ import (
 	"fmt"
 	"github.com/gonuts/commander"
 	"os"
+	"os/exec"
 	"path/filepath"
+	"strconv"
+	"strings"
 )
 
 const (
 	todo_filename = ".todo"
 )
-
+func get_tty_width() int {
+	cmd := exec.Command("stty","size")
+	cmd.Stdin = os.Stdin
+	o,err := cmd.Output()
+	if err != nil {
+		fmt.Printf("err: %s",err);
+		os.Exit(-1);
+	}
+	parts := strings.Split(strings.Trim(string(o),"\n")," ")
+	cp,_ := strconv.Atoi(parts[1])
+	return cp;
+}
+func print_header() {
+	h := "\n Todo - %s\n";
+	if os.Getenv("TODO_HEADER") != "" {
```

Comment:
I do not like use environment variable for headers. I prefer to use configuration file ex: ~/.config/todo/config.json